pages/latest/purchase.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <section v-if="buyNFTEvents.length">
+    <h2>Recent NFT Purchases</h2>
+    <table>
+      <thead>
+        <tr>
+          <th>#</th>
+          <th>NFT information</th>
+          <th>Purchase Price</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="item, i in buyNFTEvents">
```

Comment:
Should provide a unique key attribute for each item
```suggestion
        <tr v-for="item, i in buyNFTEvents" :key=“item.nft_id”>
```